### PR TITLE
IIIF Bundle Zip file export

### PR DESCRIPTION
**Pull request by benwbrum**
Downgrade to oa:FragmentSelector to display annotations in standard Mirador.

In order to export IIIF manifests that contain Readux annotations visible in
out-of-the-box Mirador installations, we simplify the annotation `selector`
to an object containing an `oa:FragmentSelector` in lieu of an array of
choice objects with bad SVG.

### CODE REVIEW

**Comment by benwbrum** [comment]
This needs further discussion before merging into `develop`.

**Comment by saracarl** [comment]
Changes needed:
1)  remove test.txt from the download zip.
2)  Can we pretty print the json so it's slightly-human-readable?
3)  Some of these are empty -- for example 
```{  
   "@context":"http://iiif.io/api/presentation/2/context.json",
   "@id":"https://0.0.0.0:3000/iiif/readux:t9vft/list/32862204.5243.emory.edu$7",
   "@type":"sc:AnnotationList",
   "resources":[  

   ]
}  
```
(That's the 7th annotation on the Fouilles export; there's no content in the annotation list.  25 of the 39 annotations look like this.  Find them by doing a ls -Sl and looking for the smallest ones.)

more to come.

**Comment by saracarl** [comment]
Per the story:
* User's name and export date are exported as metadata for the IIIF manifest.
* Should be a button, not a link. 
* Button should say "export a IIIF bundle"
* Button should have a hover explanation.  I'm not sure what friendly language is for this, perhaps "for preservation purposes"?

We need to determine if we should include level 0 images or not.

**Comment by benwbrum** [comment]
This feature is ready to be reviewed and tested.

**Comment by saracarl** [comment]
Only users who have edited the volume should see the "export" button.

**Comment by saracarl** [comment]
Button text should become "Export" or "Export/Publish" and then take you to a page that lets you choose export options -- iiif bundle, jekyll, etc.

File: apps/iiif/annotations/models.py [hunk 1]
```diff
@@ -155,6 +155,6 @@ def set_span_element(sender, instance, **kwargs):
         instance.svg = instance.oa_annotation['on'][0]['selector']['item']['value']
         instance.oa_annotation['annotatedBy'] = {'name': 'Me'}
         instance.content = instance.oa_annotation['resource'][0]['chars']
-        instance.resource_type = Annotation.COMMENTING
+        instance.resource_type = instance.oa_annotation['resource'][0]['@type']
```
Comment (benwbrum) [inline apps/iiif/annotations/models.py]: This gives us a resource type which Mirador has sent us, which is `dctypes:Text`.  While this is better than `oa:commenting`, it's still not `cnt:contentAsText`.  To complicate matters, the [relevant Web Annotation Data Model example](https://www.w3.org/TR/annotation-model/#embedded-textual-body) has standardized on `TextualBody`, as apparently the W3C standards committee [rejected Open Annotation's `contentAsText`](https://www.w3.org/TR/annotation-model/#changes-from-the-open-annotation-draft).

So this fix implements Mirador's functionality, which does not match Readux's sample data, neither of which match the WebAnnotations spec.  I suggest that we make sure this works in Readux's Mirador fork and in out-of-the-box Mirador, and call it "good" if it does.  For extra credit, we could translate both `cnt:contentAsText` and `dctypes:Text` to the new standard `TextualBody` if it works in both environments.  

cc @jayvarner @saracarl

File: apps/iiif/serializers/annotation.py [hunk 2]
```diff
@@ -40,7 +40,7 @@ def get_dump_object(self, obj):
                 "resource": {
                     "@type": obj.resource_type,
                     "format": "text/html",
-                    "chars": obj.svg,
+                    "chars": obj.content,
```
Comment (benwbrum) [inline apps/iiif/serializers/annotation.py]: One of the challenges I think we're running into is that OCR-created annotations store HTML content in `svg`, while Readux-authored annotations store HTML content in `content`.  I switched line 43 to `obj.svg` in an earlier commit to get the fonts and styling of the HTML, but my change caused the human-created annotation contents to disappear.  

@jayvarner, what do you think would be involved in moving the OCR-generated HTML from the `svg` column to the `content` column in `annotations_annotation` and the sample data?

Comment (jayvarner) [reply apps/iiif/serializers/annotation.py]: That's good. It was not my intention to retain the svg label. Now seems like a good time to make the change.

**Pull request merged**